.claude/skills/bear-cache-strategy/SKILL.md

```
@@ -0,0 +1,316 @@
+| 集計・計算 | Analytics, Report, Stats |
+| 書き込み操作 | POST/PUT/DELETE |
+
+**適用属性:**
+```php
+#[Cacheable(expirySecond: 60)]  // 短時間キャッシュ
+// または属性なし（キャッシュしない）
+```
+
+## 判定フロー
+
+```
```

markdownlint-cli2 (0.18.1)

Fenced code blocks should have a language specified

(MD040, fenced-code-language)